CDK/Linux/hello-world/lib/aws-image-builder-stack.ts

```diff
@@ -97,7 +97,7 @@ export class ImageBuilderStack extends Stack {
     for (const component of components) {
       // test if component is a valid AWS managed component arn
       const arnMatch = component.match(
-        /^arn:aws:imagebuilder:[a-z0-9\-]*:aws:component\/([\w\-]*)\/.*/
+        /^arn:aws:imagebuilder:[a-z0-9\-]*:aws-?[a-z0-9\-]*:component\/([\w\-]*)\/.*/
```

Comment:
Unfortunately this change is on the wrong `aws` string.

It needs to be on the one after `arn`, you could use: `arn:(?:aws|aws-cn|aws-us-gov|aws-iso|aws-iso-b):imagebuilder:......`

Reply:
Oops. Thanks for spotting this mistake. I have fixed the regex. I was trying to use the wildcard to fit more possible future combination of the partition values, but I have updated to use yours. I think we can always update the code when there is any new partition.